.github/workflows/dbt_slim_ci.yml

```
@@ -3,6 +3,8 @@ name: dbt slim ci (in beta)
 
 on:
   workflow_dispatch:
+  pull_request:
```

```suggestion
  pull_request:
```
  pull_request:
      paths-ignore:
        - 'deprecated-dune-v1-abstractions/**'